dist/main.js

```diff
@@ -0,0 +1,37 @@
+const renderer = new Renderer()
+const api = new APIManager(renderer)
+
+const loadPage = function()
+{
+    api.getDataFromDB()
```

Comment:
Call it instead `getCities` it describes the method better and makes reading your code easier